src/Microsoft.Data.SqlClient/netfx/src/BID/INC/CS/BidPrivateBase.cs

```diff
@@ -131,1139 +55,58 @@ internal static bool IsOn(ApiGroup flag)
         return (modFlags & flag) != 0;
     }
 
-#if BID_USE_ALL_APIGROUP
-    internal static bool AreOn(ApiGroup flags) {
-        return (modFlags & flags) == flags;
-    }
-#endif
-
-
-    private static IntPtr __noData;
+    private static IntPtr __noData = (IntPtr)(-1);
 
     internal static IntPtr NoData
     {
         get { return __noData; }
```

Comment:
Should remove this as well? __noData no long declared.

Reply:
__noData is declared on line 58:
private static IntPtr __noData = (IntPtr)(-1);
I kept NoData since it is used to set the value of an out variable in a bunch of methods.